stdlib/LLD_jll/Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "LLD_jll"
 uuid = "d55e3150-da41-5e91-b323-ecfd1eec6109"
-version = "16.0.6+4"
+version = "17.0.6+3"
```

Comment:
Is the patch version really the same?

Reply:
Yes

Reply:
17.0.6 is the latest patch release with `LLVM_VERSION_MAJOR == 17`.

It seems 6 patch releases is the standard schedule:
https://llvm.org/docs/HowToReleaseLLVM.html#annual-release-schedule
Although with v18, they've switched to `LLVM_VERSION_MINOR == 1` instead of `0`.

Reply:
Actually yes: the branch https://github.com/JuliaLang/llvm-project/commits/julia-release/17.x/ builds on top of https://github.com/JuliaLang/llvm-project/commit/6009708b4367171ccdbf4b5905cb6a803753fe18, which is the tip of https://github.com/llvm/llvm-project/releases/tag/llvmorg-17.0.6